appinfo/routes.php

```diff
@@ -339,5 +339,31 @@
 				'apiVersion' => 'v2(\.[1-3])?'
 			]
 		],
+		// Submissions linking with file in cloud
+		[
+			'name' => 'api#linkFile',
+			'url' => '/api/{apiVersion}/form/link/{fileFormat}',
+			'verb' => 'POST',
+			'requirements' => [
+				'apiVersion' => 'v2(\.3)?',
```

Comment:
```suggestion
				'apiVersion' => 'v2.3',
```

The endpoints are only available in v2.3, not in v2. Same for the other two... 🙂

Reply:
it not works for me with your suggested change

![image](https://github.com/nextcloud/forms/assets/191082/0c45f543-983e-4467-98c0-ed496a369507)

Reply:
You have to adjust the call in results.vue too ;)